libcloud/test/dns/test_nsone.py

```diff
@@ -0,0 +1,254 @@
+import sys
+import unittest
+
+from libcloud.test import MockHttp
+from libcloud.test.file_fixtures import DNSFileFixtures
+from libcloud.test.secrets import DNS_PARAMS_NSONE
```

Comment:
@tonybaloney I  updated and pushed secrets.py. I do not know why the travis builds are failing. The tests pass when run on my local machine.